flow/src/main/java/com/sequenceiq/flow/reactor/config/EventBusConfig.java

```diff
@@ -48,13 +54,13 @@ private Dispatcher getEventBusDispatcher() {
         ClassLoader context = new ClassLoader(Thread.currentThread()
                 .getContextClassLoader()) {
         };
-        MDCCleanerThreadPoolExecutor executorService = new MDCCleanerThreadPoolExecutor(eventBusThreadPoolSize,
-                eventBusThreadPoolSize,
+        MDCCleanerThreadPoolExecutor executorService = new MDCCleanerThreadPoolExecutor(eventBusThreadPoolCoreSize,
+                eventBusThreadPoolMaxSize,
                 0L,
                 TimeUnit.MILLISECONDS,
-                new LinkedBlockingQueue<>(eventBusThreadPoolSize),
+                new LinkedBlockingQueue<>(eventBusThreadPoolBacklogSize),
                 new NamedDaemonThreadFactory("reactorDispatcher", context),
                 (r, executor) -> r.run());
```

Comment:
I think this time we should define a proper `RejectedExecutionHandler` here to be able to track the lack of threads from the logs.

Reply:
replaced with logging the error